lib/spack/spack/test/gcs_fetch.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+import pytest
+
+import spack.config as spack_config
+import spack.fetch_strategy as spack_fs
+import spack.stage as spack_stage
```

Comment:
Can we use the fully qualified names:
```suggestion
import spack.config
import spack.fetch_strategy
import spack.stage
```
? For the first and lat modules it won't even be more verbose, it's just `.` instead of `_`

Reply:
Definitely. :)